modules/live-schedule-impl/src/main/java/org/opencastproject/liveschedule/impl/LiveScheduleServiceImpl.java

```diff
@@ -229,9 +229,24 @@ protected void activate(ComponentContext context) {
   public boolean createOrUpdateLiveEvent(String mpId, DublinCoreCatalog episodeDC) throws LiveScheduleException {
     MediaPackage mp = getMediaPackageFromSearch(mpId);
     if (mp == null) {
+      // Check if capture not over. We have to check because we may get a notification for past events if
+      // the admin ui index is rebuilt
+      DCMIPeriod period = EncodingSchemeUtils.decodeMandatoryPeriod(episodeDC.getFirst(DublinCore.PROPERTY_TEMPORAL));
+      if (period.getEnd().getTime() < System.currentTimeMillis()) {
```

Comment:
Not that it's very relevant but this should be `<=` since the end date-time should be non-inclusive.